DscResources/MSFT_GroupResource/MSFT_GroupResource.psm1

```diff
@@ -91,11 +95,12 @@ function Get-TargetResource
     [CmdletBinding()]
     param
     (
-        [Parameter(Mandatory = $true)]
+        [Parameter(Mandatory = $true, HelpMessage="The name of the group to create, modify, or remove.")]
```

Comment:
HelpMessage properties should be added to the parameters of SetTargetResource instead of GetTargetResource